content/recipes/interoperability/ontology-operations-tools.md

```diff
@@ -273,410 +55,138 @@ The table below is an overview of ontology strategies tools identified. Details
+| [Vapur](https://bio.tools/vapur)| A Search Engine to Find Related Protein.Vapur is an online entity-oriented search engine for the COVID-19 anthology. Vapur is empowered with a semantic inverted index that is created through named entity recognition and relation extraction on CORD-19 abstracts. |N/A| Pathology, Ontology and terminology, Natural language processing, Enzymes |Python|
+| [matscholar](https://bio.tools/matscholar)| A Python library for materials-focused natural language processing (NLP). Named Entity Recognition and Normalization Applied to Large-Scale Information Extraction from the Materials Science Literature.|[MIT](https://spdx.org/licenses/MIT)| Chemistry, Ontology and terminology, Natural language processing | Command-line tool |
+|[CollaboNet](https://bio.tools/collabonet)| Collaboration of deep neural networks for biomedical named entity recognition.|[MIT](https://spdx.org/licenses/MIT)| Ontology and terminology, Natural language processing, Machine learning | Command-line tool |
+| [Calchas](https://bio.tools/calchas)| A web based framework that takes advantage of domain specific ontologies, and Natural Language Processing, aiming to empower exploration of biomedical resources via semantic-based querying and search. The NLP engine analyzes the input free-text query and translates it into targeted queries with terms from the underlying ontology. | N/A | Medical informatics, Ontology and terminology, Natural language processing, Bioinformatics| Web application   |
+|[QTL TableMiner++(QTM)](https://bio.tools/QTM) | It is a command-line tool to retrieve and semantically annotate results obtained from QTL mapping experiments. It takes full-text articles from the Europe PMC repository as input and outputs the extracted QTLs into a relational database (SQLite) and text file (CSV).|[Apache-2.0](https://spdx.org/licenses/Apache-2.0)| Ontology and terminology| Command-line tool |
+| [thbp](https://bio.tools/thbp)| Mapping anatomical related entities to human body parts based on wikipedia in discharge summaries|N/A| Anatomy, Ontology and terminology, Natural language processing||
 
 ### Ontology mapping
 
 The process of determining correspondences between equivalent concepts in alternative ontologies, and other vocabularies. This may include mapping to convey different levels of granularity.
 
-__Manually curated list of tools__
+__Curated tools__
+
+|Tool|Description|License|Topics|Resource Type|How to use|
+|---|--|--|--|--|--|
+|[OxO](https://www.ebi.ac.uk/spot/oxo/index)|A service for finding mappings (or cross-references) between terms from ontologies, vocabularies and coding standards. |[EMBL-EBI Terms of Use](https://www.ebi.ac.uk/about/terms-of-use/)|Ontology alignment| GUI and API| {ref}`ontology-alignment-oxo` |
```

Comment:
How-To reference missing?

Reply:
the oxo recipe will not be available for now. the ref is removed